be/src/olap/tablet_manager.cpp

```diff
@@ -555,8 +552,8 @@ TabletSharedPtr TabletManager::get_tablet(TTabletId tablet_id, SchemaHash schema
         if (!tablet->is_used()) {
             OLAP_LOG_WARNING("tablet cannot be used. [tablet=%ld]", tablet_id);
             tablet.reset();
-        } else if (load_tablet && !tablet->is_loaded()) {
-            if (tablet->load() != OLAP_SUCCESS) {
+        } else if (load_tablet && !tablet->init_success()) {
```

Comment:
I think is_inited is a better name?

Reply:
I think init_succeeded may be better to describe it.
is_inited can not differentiate success or fail literally